forum/views.py

```diff
@@ -24,13 +23,7 @@ class Meta:
         fields = ('content',)
 
 
-def require_login(function):
```

Comment:
Don't remove the require login, probably need to merge with master first before PR